packages/elastic-safer-lodash-set/scripts/patches/_baseSet.js.patch

```diff
@@ -0,0 +1,31 @@
+1,5c1,15
+< var assignValue = require('./_assignValue'),
+<     castPath = require('./_castPath'),
+<     isIndex = require('./_isIndex'),
+<     isObject = require('./isObject'),
+<     toKey = require('./_toKey');
+---
+> /*
+>  * This file is forked from the lodash project (https://lodash.com/),
+>  * and may include modifications made by Elasticsearch B.V.
+>  * Elasticsearch B.V. licenses this file to you under the MIT License.
+>  * See `packages/elastic-safer-lodash-set/LICENSE` more information.
```

Comment:
__note:__ we need to regenerate patch I believe (license text changed).

Reply:
Fixed 👍